.github/workflows/generate-pinval.yaml

```diff
@@ -38,6 +38,7 @@ on:
 
 env:
   PYTHONUNBUFFERED: "1"
+  UV_SYSTEM_PYTHON: 1
```

Comment:
This is a common pattern we follow in other uv workflows ([see here for an example](https://github.com/ccao-data/data-architecture/blob/6f725f3851947480f368bb84bfd4cee963f18d21/.github/workflows/socrata_upload.yaml#L33)) to make it easier to install uv dependencies. This var tells uv to install packages globally using the system Python installation, which means we don't need to create or manage virtualenvs. This works well in a CI environment, where we spin up an isolated container for each run, but it's not recommended for local development because it can make it harder to enforce separation between projects. See [the docs](https://docs.astral.sh/uv/reference/environment/#uv_system_python) for more details.

Reply:
Interesting, that's a nice feature